config/crd/bases/enterprise.splunk.com_clustermasters.yaml

```diff
@@ -1151,6 +1151,19 @@ spec:
                 - Always
                 - IfNotPresent
                 type: string
+              imagePullSecrets:
+                description: Sets imagePullSecrets if image is being pulled from a
+                  private registry See https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/
+                items:
+                  description: LocalObjectReference contains enough information to
+                    let you locate the referenced object inside the same namespace.
+                  properties:
+                    name:
+                      description: 'Name of the referent. More info: https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#names
+                        TODO: Add other useful fields. apiVersion, kind, uid?'
```

Comment:
Where's this TODO come from?  Is that from the kubernetes code?

Reply:
Yes it was automatically added while generating CRDs(possible from the source code of kubernetes)